readthedocs/templates/base.html

```diff
@@ -105,76 +105,97 @@
         {% block footer-content %}
 
         <div class="column-logo">
-          <div class="footerlogo"></div>
+          <div id="email-signup">
+            <form action="https://readthedocs.us3.list-manage.com/subscribe/post?u=a6a22369cc2b356379cf789ca&amp;id=a85a83a5a5" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate" target="_blank" novalidate>
+              <div>
+                <h4>{% trans 'Stay Updated' %}</h4>
+
+                <ul>
+                  <li>
+                    <a href="https://blog.readthedocs.com/">{% trans 'Blog' %}</a>
+                  </li>
+                </ul>
+
+                <p>{% trans 'Sign up for our newsletter to get our latest blog updates delivered to your inbox weekly.' %}</p>
```

Comment:
I might also link the `blog` here to the blog, and perhaps even more it from above. That might make it a bit more seemless.